applications/nrf_desktop/configuration/nrf52833dk_nrf52820/prj.conf

```diff
@@ -0,0 +1,179 @@
+CONFIG_DESKTOP_BLE_SCAN_DURATION_S=30
+
+CONFIG_DESKTOP_BLE_DISCOVERY_ENABLE=y
+
+CONFIG_DESKTOP_CONFIG_CHANNEL_ENABLE=y
+
+################################################################################
+# Zephyr Configuration
+
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=1536
+CONFIG_ISR_STACK_SIZE=1280
+CONFIG_MAIN_STACK_SIZE=768
+CONFIG_IDLE_STACK_SIZE=128
+CONFIG_BT_RX_STACK_SIZE=2048
+CONFIG_BT_HCI_TX_STACK_SIZE_WITH_PROMPT=y
+CONFIG_BT_HCI_TX_STACK_SIZE=1536
+
+CONFIG_BOOT_BANNER=n
+
+CONFIG_NUM_COOP_PRIORITIES=10
+CONFIG_NUM_PREEMPT_PRIORITIES=11
+CONFIG_TIMESLICING=n
+
+CONFIG_HEAP_MEM_POOL_SIZE=1024
+
+CONFIG_SYS_CLOCK_TICKS_PER_SEC=1000
+CONFIG_SYSTEM_CLOCK_NO_WAIT=y
+
+CONFIG_PM_POLICY_APP=y
+
+CONFIG_HW_STACK_PROTECTION=y
+CONFIG_RESET_ON_FATAL_ERROR=n
+
+CONFIG_GPIO=y
+
+CONFIG_HWINFO=y
+
+CONFIG_FLASH=y
+CONFIG_FLASH_PAGE_LAYOUT=y
+CONFIG_FLASH_MAP=y
+CONFIG_NVS=y
+
+CONFIG_SETTINGS=y
+CONFIG_SETTINGS_DYNAMIC_HANDLERS=n
+CONFIG_SETTINGS_NVS=y
+
+CONFIG_REBOOT=y
+
+CONFIG_SPEED_OPTIMIZATIONS=y
+
+CONFIG_LED=y
+CONFIG_LED_GPIO=y
+
+CONFIG_USB_NRFX=y
+CONFIG_USB_NRFX_EVT_QUEUE_SIZE=16
+CONFIG_USB_DEVICE_STACK=y
+CONFIG_USB_DEVICE_MANUFACTURER="Nordic Semiconductor ASA"
+CONFIG_USB_DEVICE_PRODUCT="Dongle nRF52 Desktop"
+CONFIG_USB_DEVICE_VID=0x1915
+CONFIG_USB_DEVICE_PID=0x52E1
+CONFIG_USB_DEVICE_REMOTE_WAKEUP=y
+CONFIG_USB_DEVICE_HID=y
+#CONFIG_USB_DEVICE_LOG_LEVEL_OFF=y
+#CONFIG_USB_DRIVER_LOG_LEVEL_OFF=y
+CONFIG_USB_HID_POLL_INTERVAL_MS=1
+CONFIG_USB_HID_DEVICE_COUNT=2
+
+CONFIG_BT=y
+CONFIG_BT_SETTINGS=y
+CONFIG_BT_SMP=y
+CONFIG_BT_SIGNING=y
+CONFIG_BT_MAX_PAIRED=2
+CONFIG_BT_MAX_CONN=2
+CONFIG_BT_PRIVACY=y
+CONFIG_BT_LL_SW_SPLIT=y
+
+CONFIG_BT_DEVICE_NAME="Dongle nRF52 Desktop"
+CONFIG_BT_DEVICE_APPEARANCE=962
+
+CONFIG_BT_CTLR=y
+CONFIG_BT_CTLR_CONN_PARAM_REQ=n
+CONFIG_BT_CTLR_TX_PWR_0=y
+CONFIG_BT_BUF_ACL_TX_SIZE=35
+CONFIG_BT_CTLR_DATA_LENGTH_MAX=35
+CONFIG_BT_L2CAP_TX_BUF_COUNT=4
+
+CONFIG_BT_CENTRAL=y
+
+CONFIG_BT_SCAN=y
+CONFIG_BT_SCAN_FILTER_ENABLE=y
+CONFIG_BT_SCAN_NAME_CNT=2
+CONFIG_BT_SCAN_ADDRESS_CNT=2
+
+CONFIG_BT_GATT_CLIENT=y
+CONFIG_BT_SETTINGS_CCC_STORE_ON_WRITE=y
+CONFIG_BT_SETTINGS_CCC_LAZY_LOADING=n
+CONFIG_BT_GATT_DM=y
+
+CONFIG_BT_HOGP=y
+CONFIG_BT_HOGP_REPORTS_MAX=12
+
+CONFIG_ENTROPY_CC3XX=n
+
+CONFIG_EVENT_MANAGER=y
+CONFIG_LINKER_ORPHAN_SECTION_PLACE=y
+
+################################################################################
+
+CONFIG_ASSERT=y
+CONFIG_ASSERT_LEVEL=2
+
+
+CONFIG_USE_SEGGER_RTT=n
+CONFIG_SERIAL=n
+CONFIG_CONSOLE=n
+CONFIG_UART_CONSOLE=n
+
+
+#CONFIG_LOG=y
+#CONFIG_LOG_MAX_LEVEL=1
+#CONFIG_LOG_DEFAULT_LEVEL=1
+#CONFIG_LOG_MODE_IMMEDIATE=y
+#CONFIG_LOG_BACKEND_RTT=y
+#CONFIG_LOG_BACKEND_RTT_MODE_DROP=y
+#CONFIG_LOG_PRINTK=y
+#CONFIG_LOG_BACKEND_RTT_MESSAGE_SIZE=32
+#CONFIG_LOG_BACKEND_SHOW_COLOR=n
+#CONFIG_LOG_BACKEND_FORMAT_TIMESTAMP=n
+
+#CONFIG_CONSOLE=y
+#CONFIG_USE_SEGGER_RTT=y
+#CONFIG_SEGGER_RTT_BUFFER_SIZE_UP=256
+#CONFIG_RTT_CONSOLE=y
+#CONFIG_UART_CONSOLE=n
```

Comment:
I believe its leftover. Remove this.

Reply:
It is the same as in nrf52820dongle_nrf52820 on which I based my code. Thought it is there for a reason.

Reply:
This is a mistake, 100%. Please remove this.
If you could, prepare another PR for removing this from `nrf52820dongle_nrf52820` too.

Reply:
I will do it in this PR only in separate commit